packs/databricks/dashboards/databricks.json

```diff
@@ -0,0 +1,427 @@
+{
+    "name": "Spark - Overview v2",
+    "description": "",
+    "pages": [
+      {
+        "name": "Spark - Overview",
+        "description": "",
+        "widgets": [
+          {
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "layout": {
+              "column": 1,
+              "row": 1,
+              "height": 2,
+              "width": 1
+            },
+            "title": "",
+            "rawConfiguration": {
+              "text": "\n![Add Images](https://d2.alternativeto.net/dist/icons/apache-spark_147518.png?width=128&height=128&mode=crop&upscale=false)"
+            },
+            "linkedEntityGuids": null
+          },
+          {
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "layout": {
+              "column": 2,
+              "row": 1,
+              "height": 2,
+              "width": 1
+            },
+            "title": "Spark Application Count",
+            "rawConfiguration": {
+              "nrqlQueries": [
+                {
+                  "accountId": 1606862,
```

Comment:
this needs to be set to 0, for all occurrences of accountId

Reply:
fixed this